Vagrantfile

```diff
@@ -61,6 +61,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   # Make the project root available to the guest VM.
   # config.vm.synced_folder '.', '/vagrant'
+  config.vm.synced_folder '.', '/vagrant'
+  config.vm.synced_folder '~/vagrantResources/', '/published'
```

Comment:
let me remove those.

Thanks for reviewing it. I'm quite new to vagrant and virtualbox. Please feel free to refactor everything here.